src/main/java/frc/robot/RobotContainer.java

```diff
@@ -6,13 +6,20 @@
 
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
-
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.commands.drive.DriveCommand;
 public class RobotContainer {
+  CommandXboxController driverController = new CommandXboxController(4);
+  Trigger driverAButton = driverController.a();
   public RobotContainer() {
     configureBindings();
   }
 
-  private void configureBindings() {}
+  private void configureBindings() {
+    driverAButton.onTrue(new DriveCommand(null, null, null, null, null, driverAButton, driverAButton));
```

Comment:
why are we passing in null to this?

Reply:
bro it was hard enough to get @mychenezpz to write a button binding, id be proud of him for doing that lol. its null bc he didnt get to it ig

Reply:
ill write some better examples when i have time tmrw